### ISSUE TITLE
Add a "copy to clipboard" button to results screen

### ISSUE THREAD

**Issue by nahojjjen** [farkam135/GoIV#532]
A button which allows the user to force GoIV to place info in the clipboard, even if the setting is off.

Issue derived from #511

**Comment by sarav** [comment]
When you do this, can you make sure this only shows up if the copy to clipboard option is off?

Also, I wonder if a better approach instead of a button would be to click on a particular row of there are multiple IV combinations so that the user has more control of which combination gets copied? And then just tap the shown IV values if it's only one combination?

Only problem with this is that is won't be obvious to the user. But this is a niche feature (wanting clipboard option off, but still wanting to copy) so maybe avoiding UI clutter at the expense of not being obvious is okay?

**Comment by TripSixes** [comment]
I'll add my +1 for the ability to select a row (when multiple) to copy that info into the clipboard.  There are MANY instances that this feature would be useful.
1) You and a friend catch same pokemon.  Either you or your friend GoIV shows single result, while the other shows multiple (even after appraisal).  This would allow the multiple-result user to select the correct one.
2) There are many times that I want to see the final overview screen again for a pokemon that I've already calculated (mainly to forecast evolution CP and candy/stardust cost). If that Pokemon was one that had multiples pre-appraisal and single after appraisal, then I am forced to select the appraisals again to get back to the single result.  Having ability to "select" the correct row from the multiple table would be more efficient that re-entering appraisal info.

.. hmm that's all I got for now.
